product_packaging_dimension_decimal/views/product_packaging.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Allow user to record float value -->
+    <record id="product_packaging_measure_float_value" model="ir.ui.view">
+        <field name="name">Product Packaging: changing to float field</field>
+        <field name="model">product.packaging</field>
+        <field name="inherit_id" ref="delivery.product_packaging_delivery_form"/>
+        <field name="arch" type="xml">
+
+            <!-- Hide the default uom char field instead of replacing it ...-->
+            <xpath expr="//div[@name='height']/span/field[@name='length_uom_name']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//div[@name='width']/span/field[@name='length_uom_name']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//div[@name='packaging_length']/span/field[@name='length_uom_name']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+
+            <!-- ... then add the many2one uom field -->
+            <xpath expr="//div[@name='height']/span/field[@name='length_uom_name']" position="before">
+                <field name="height_uom_id" nolabel="1" style="width:25%%" />
```

Comment:
Lors de la sauvegarde du formulaire `Delivery Packages`, on remarque que l'unité de mesure n'est pas aligné avec les champs height, width and length.